src/pseudopeople/utilities.py

```diff
@@ -166,6 +167,38 @@ def get_state_abbreviation(state: str) -> str:
         raise ValueError(f"Unexpected state input: '{state}'") from None
 
 
+####################
+# Engine utilities #
+####################
+
+
+@dataclass
+class Engine:
+    name: str
+
+
+PANDAS = Engine("pandas")
+MODIN = Engine("modin")
+
+
+def get_engine_from_string(engine: str):
+    if engine == "pandas":
+        return PANDAS
+    elif engine == "modin":
+        return MODIN
+    else:
+        raise ValueError(f"Unknown engine {engine}")
+
+
+try:
+    # Optional dependency
+    import modin.pandas as mpd
+
+    DATAFRAME = Union[mpd.DataFrame, pd.DataFrame]
+except ImportError:
+    DATAFRAME = pd.DataFrame
```

Comment:
When we've had type aliases in vivarium, we haven't used all-caps, so perhaps this ought to be `DataFrame` rather than `DATAFRAME`?